Gems/Multiplayer/Code/Source/NetworkEntity/NetworkEntityAuthorityTracker.h

```diff
@@ -29,37 +29,13 @@ namespace Multiplayer
         HostId GetEntityAuthorityManager(ConstNetworkEntityHandle entityHandle) const;
 
     private:
-
-        HostId GetEntityAuthorityManagerInternal(ConstNetworkEntityHandle entityHandle) const;
-
         NetworkEntityAuthorityTracker& operator= (const NetworkEntityAuthorityTracker&) = delete;
 
-        struct TimeoutData final
-        {
-            TimeoutData() = default;
-            TimeoutData(ConstNetworkEntityHandle entityHandle, const HostId& previousOwner);
-            ConstNetworkEntityHandle m_entityHandle;
-            HostId m_previousOwner = InvalidHostId;
-        };
-
-        struct NetworkEntityTimeoutFunctor final
-            : public AzNetworking::ITimeoutHandler
-        {
-            NetworkEntityTimeoutFunctor(NetworkEntityAuthorityTracker& networkEntityAuthorityTracker, INetworkEntityManager& m_networkEntityManager);
-            AzNetworking::TimeoutResult HandleTimeout(AzNetworking::TimeoutQueue::TimeoutItem& item) override;
-        private:
-            AZ_DISABLE_COPY_MOVE(NetworkEntityTimeoutFunctor);
-            NetworkEntityAuthorityTracker& m_networkEntityAuthorityTracker;
-            INetworkEntityManager& m_networkEntityManager;
-        };
-
-        using TimeoutDataMap = AZStd::unordered_map<NetEntityId, TimeoutData>;
+        using TimeoutDataMap = AZStd::unordered_set<NetEntityId>;
```

Comment:
TimeoutDataMap and m_timeoutDataMap are pretty confusing name wise now. Maybe TimeoutNetEntities and m_timeoutNetIds?